alns/select/MABSelector.py

```diff
@@ -0,0 +1,216 @@
+import itertools
+from typing import Callable, List, Optional, Tuple, Union
+
+import numpy as np
+import pandas as pd
+from mabwiser.mab import MAB
+from mabwiser.utils import Num
+from numpy.random import RandomState
+
+from alns.State import State
+from alns.select.OperatorSelectionScheme import OperatorSelectionScheme
+
+
+class MABSelector(OperatorSelectionScheme):
+    """
+    A selector that uses any multi-armed-bandit algorithm from MABWiser.
+
+    This selector is a wrapper around the many multi-armed bandit algorithms
+    available in the `MABWiser <https://github.com/fidelity/mabwiser>`_
+    library. Since ALNS operator selection can be framed as a
+    multi-armed-bandit problem, this wrapper allows you to use a variety of
```

Comment:
may be one word about the connection between bandits to alns about "tuples as arms"